agents/PCS-Large-Exhange-Rate-Update/src/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { ethers, Finding, FindingType, FindingSeverity } from "forta-agent";
+import { BigNumber } from "ethers";
+import { getCreate2Address } from "@ethersproject/address";
+
+const getPancakePairCreate2Address = (
+  pancakeFactoryAddr: string,
+  token0: string,
+  token1: string,
+  initCode: string
+): string => {
+  const salt = ethers.utils.solidityKeccak256(["address", "address"], [token0, token1]);
+  return getCreate2Address(pancakeFactoryAddr, salt, initCode);
+};
+
+const createFinding = (exchangerateOld: Number, exchangerateNew: Number): Finding => {
+  return Finding.from({
+    name: "Large exchangeRate change",
+    description: `Old exchange rate ${ethers.utils.formatEther(
+      exchangerateOld.toString()
+    )} & ${ethers.utils.formatEther(exchangerateNew.toString())}`,
```

Comment:
```suggestion
    description: `Old exchange rate ${ethers.utils.formatEther(
      exchangerateOld.toString()
    )} & New exchange rate ${ethers.utils.formatEther(exchangerateNew.toString())}`,
```
Can you also add, by what percentage change tracked?

Reply:
Done